### DISCUSSION TITLE
问大佬vercel部署时卡住不动怎么解决

### DISCUSSION THREAD

**Discussion by RainHoutas** [walinejs/waline#1224] [Q&A] [answered]
![屏幕截图 2022-07-23 110308](https://user-images.githubusercontent.com/97776439/180588215-88f149cf-0c42-4b57-bd79-2b73ca90ca97.png)
库已经创建成功了
卡在这个界面一动不动，试了好几次，问大佬该怎么办啊

**Comment by lizheming** [comment, marked as answer]
我试了一下没有问题，看看切换下网络试试。不行的话可以给 Vercel 提工单

**Reply by RainHoutas** [reply]
谢谢大佬，我再看看
